src/legacy/ui/public/chrome/api/theme.ts

```diff
@@ -19,8 +19,10 @@
 
 import * as Rx from 'rxjs';
 
-import { Brand, ChromeStart } from '../../../../../core/public/chrome';
+import { Chrome, CoreStart } from '../../../../../core/public/';
 
+type Brand = Chrome['Brand'];
+type ChromeStart = CoreStart['chrome'];
```

Comment:
I'd rather add all start contract as direct export in `core/public/` for such cases. WDYT?

Reply:
I've also been thinking about this a lot today. It feels like an API that gets consumed by external parties should expose all of it's types at the top-level (perhaps divided in namespaces), developers shouldn't have to dig around inside other types to find the types they need.

Specifically accessing the `ChromeStart` type from a property called `[chrome]` isn't as intuitive as maybe:

```
import {ChromeStart} from '..../core/public'
let newPlatformChrome: ChromeStart;
``` 
or 
```
import {Chrome} from '..../core/public'
let newPlatformChrome: Chrome.ChromeStart;
```

Reply:
https://github.com/elastic/kibana/pull/33562#discussion_r267730546